documentation/4.0/content/managing-domains/model-in-image/usage.md

```diff
@@ -187,11 +187,9 @@ that are copied from [Auxiliary Images]({{<relref "/managing-domains/model-in-im
 
  - There are additional attributes that are common to all domain home source types, such as the `image` field. See the Domain Resource [schema](https://github.com/oracle/weblogic-kubernetes-operator/blob/{{< latestMinorVersion >}}/documentation/domains/Domain.md) and [documentation]({{< relref "/managing-domains/domain-resource.md" >}}) for a full list of Domain fields.
 
- - There are also additional fields that are specific to JRF domain types. For more information, see [Requirements for JRF domain types](#requirements-for-jrf-domain-types).
-
  - For fully specified Model in Image Domain YAML file examples,
    see the [`kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources`](https://github.com/oracle/weblogic-kubernetes-operator/tree/{{< latestMinorVersion >}}/kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources) GitHub directory for the [Model in Image sample]({{< relref "/samples/domains/model-in-image/_index.md" >}}).
-   The `WLS` and `JRF` subdirectories in this directory correspond to the `configuration.model.domainType`.
+   The `WLS` subdirectory in this directory corresponds to the `configuration.model.domainType`.
```

Comment:
@rjeberhard @rosemarymarano Will we be removing the JRF related directories? If so, then this sentence can be removed.

Reply:
Tom is correct. Let's please remove this sentence.